### PR TITLE
Move i18n build to composer, fix language path

### LINKED ISSUE #566: Build command breaks on Buddy
Better late than never :).

I don't know for sure what the best approach would be to fix this, but the setup in package.json is breaking our Buddy deploys. And so this is definitely an internal WDS issue, but anyone that uses Buddy (or a deploy tool that works similarly, with a separation between Node and PHP environments) will run into this.

## The problem ##
Right now, to build everything you can simply run `npm run build`. This will build all of the CSS, JavaScript, images, and translation files, but as a result it will also make the assumption that you have PHP and WP CLI installed on your build environment.

This will work perfectly when you do it on a local environment, but on Buddy the environment your action runs on is either Node or PHP enabled and so it _cannot_ do both of those things in the same action unless you _also_ manually install Node or PHP on that action's environment. This is super clunky and difficult to maintain.

## Possible solutions ##
So, the way I see it there are two ways to fix this. 

The first one is what I ended up doing on the project I ran into these issues with and it's to take `theme:pot` out of the general `theme` Node command and added a separate step in Buddy's deploy pipeline to build the POT file before deploying by running the same WP CLI command that `theme:pot` runs (since this environment doesn't have node, running commands from package.json is essentially out of the question).

What this does do, however, is break the "one command to build _all the things_" philosophy. And so the alternative, which is more of a compromise for that and probably the way we should go here, is to have a second node command (`npm run build:buddy` or `npm run build:node` or whatever) that will only build all the Node-buildable assets and leave the current commands alone. Obviously, this can also be reversed where `npm run build` will be the command that builds the Node-buildable assets and we add a `npm run build:all` or something that also builds POT files. I have no strong opinion either way :).

In either scenario we'll need to update the project template in Buddy to match whichever route we end up going here, but I figure this ticket can serve as the impetus for that change :).

### PR DESCRIPTION
**Pull request by jrfoell**
Closes #566 and #595 

### DESCRIPTION

- Moved the i18n make-pot action from npm to composer as it requires wp-cli & php.

### OTHER

- [x] Is this issue accessible? (Section 508/WCAG 2.0AA)
- [x] Does this issue pass all the linting? (PHPCS, ESLint, SassLint)
- [x] Does this pass CBT?

### STEPS TO VERIFY

Run a `npm run build` normally, it should do the same thing.
Run `npm run theme:node` and it should only build the npm related stuff.

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?
Possibly?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,8 +30,9 @@
     "postinstall": "npm run build",
     "start": "wds-start",
     "theme": "run-s theme:pot theme:icons",
+    "theme:node": "run-s theme:icons",
```
Comment (coreymcollins) [inline package.json:33]: Do we really need `theme:node`? With this change, its only purpose is to run `theme:icons` and if someone wants to run that they can just do `npm run theme:icons`.  cc @gregrickaby @jrfoell

Comment (coreymcollins) [reply package.json:33]: Or is `theme:node` specific to our Buddy workflow? Maybe it's to future proof for anything additional that may be added to `theme:node` down the road?

Comment (jrfoell) [reply package.json:33]: It's specifically for our Buddy workflow. We keep the normal build so the general public can build "all the things" with one command

**Comment by coreymcollins** [comment]
Tested and working on my end, want to get @gregrickaby's eyes on it before we merge it in.

**Review by coreymcollins — APPROVED** [review]

**Comment by coreymcollins** [comment]
@jrfoell – do you want to prep the updates to the Wiki so you can update that once we merge this in as well?

**Review by gregrickaby — APPROVED** [review]

**Comment by jrfoell** [comment]
Thanks @gregrickaby & @coreymcollins - I added the new commands to the documentation here: https://github.com/WebDevStudios/wd_s/wiki#npm-scripts

**Pull request merged**